bot/cogs/grades_data/grades_cog.py

```diff
@@ -277,5 +341,42 @@ async def grades(self, ctx, course, year=MIN_YEAR):
 
         await ctx.send(embed=embed)
 
+    @commands.command()
+    async def prof(self, ctx, firstName, lastName, detailed = bool('yes')):
```

Comment:
d.py exposes a boolean type hint converter for command args. relevant docs: https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#bool

Reply:
So change something?

Reply:
ya, make it a boolean converter. Thats much more flexible then a string